docs/CONTRIBUTORS.rst

```diff
@@ -1,92 +1,22 @@
-    1 Letterman Drive
-    Suite D4700
-    San Francisco, CA, 94129
-
-    Everyone is permitted to copy and distribute verbatim copies of this
-    license document, but changing it is not allowed.
-
-    Developer's Certificate of Origin 1.1
-
-    By making a contribution to this project, I certify that:
-
-    (a) The contribution was created in whole or in part by me and I have the
-    right to submit it under the open source license indicated in the file; or
-
-    (b) The contribution is based upon previous work that, to the best of my
-    knowledge, is covered under an appropriate open source license and I have
-    the right under that license to submit that work with modifications,
-    whether created in whole or in part by me, under the same open source
-    license (unless I am permitted to submit under a different license), as
-    indicated in the file; or
-
-    (c) The contribution was provided directly to me by some other person who
-    certified (a), (b) or (c) and I have not modified it.
-
-    (d) I understand and agree that this project and the contribution are
-    public and that a record of the contribution (including all personal
-    information I submit with it, including my sign-off) is maintained
-    indefinitely and may be redistributed consistent with this project or the
-    open source license(s) involved.
-
-
-To facilitate development and installation of edited version of the code base,
-developers are encouraged to install `Virtualenv <https://virtualenv.pypa.io/en/latest/index.html>`_,
-which is a tool to create isolated Python environments.  It includes
-``pip`` and ``setuptools``, Python packages that can be used to
-install TUF and its dependencies. All installation methods of
-virtualenv are outlined in the `installation
-section <https://virtualenv.pypa.io/en/latest/installation.html>`_,
-and instructions for installing locally from source are provided here:
-::
-
-    $ curl -O https://pypi.python.org/packages/source/v/virtualenv/virtualenv-15.0.3.tar.gz
-    $ tar xvfz virtualenv-15.0.3.tar.gz
-    $ cd virtualenv-15.0.3
-    $ python3 virtualenv.py myVE
-
+Contribute to python-tuf by submitting pull requests against the "develop"
+branch of this repository. Detailed instructions are available in our
+`development guidelines
+<https://github.com/secure-systems-lab/lab-guidelines/blob/master/dev-workflow.md>`_.
+All submitted code should follow our `style guidelines
+<https://github.com/secure-systems-lab/code-style-guidelines/blob/master/python.md>`_
+and must be `unit tested <#unit-tests>`_.
 
 Development Installation
 ========================
 
 To work on the TUF project, it's best to perform a development install.
 
+To facilitate development and installation of edited version of the code base,
+developers are encouraged to use `venv <https://docs.python.org/3/library/venv.html>`_.
+
 1. First, `install non-Python dependencies <INSTALLATION.rst#non-python-dependencies>`_.
```

Comment:
link is broken on RTD

Reply:
I lean towards using the absolute link. Like that it will not only render correctly on RTD, but also on GitHub, which [currently it doesn't](https://github.com/theupdateframework/python-tuf/blob/1ae438b30f015c57c4d4a2b89bb7d72d303cd6c1/docs/CONTRIBUTORS.rst#development-installation).

Reply:
I'm OK with that. I was trying to use the sphinx feature
I left it enabled in the case it is needed for label reference.